src/Phpmig/Adapter/Illuminate/Database.php

```diff
@@ -21,13 +21,13 @@ class Database implements AdapterInterface
     protected $tableName;
 
     /**
-     * @var \Illuminate\Database\Capsule\Manager
+     * @var \Illuminate\Database\Connection
```

Comment:
No, the `adapter` property is now a Connection instead of a Connection Manager and is set upon instantiation (line 30).